src/test/regress/expected/multi_generate_ddl_commands.out

```diff
@@ -48,9 +48,9 @@ CREATE TABLE table_constraint_table (
 	CONSTRAINT bids_ordered CHECK (min_bid > max_bid)
 );
 SELECT master_get_table_ddl_events('table_constraint_table');
-                                                                      master_get_table_ddl_events
+                                                                       master_get_table_ddl_events
 ---------------------------------------------------------------------
- CREATE TABLE public.table_constraint_table (bid_item_id bigint, min_bid numeric NOT NULL, max_bid numeric NOT NULL, CONSTRAINT bids_ordered CHECK (min_bid > max_bid))
+ CREATE TABLE public.table_constraint_table (bid_item_id bigint, min_bid numeric NOT NULL, max_bid numeric NOT NULL, CONSTRAINT bids_ordered CHECK ((min_bid > max_bid)))
```

Comment:
Thanks! Can you add the test cases from the linked issues as well?

Reply:
Is CHECK(TRUE) a good test case, or would you want be to add another one? I thought `inet_client_port()::boolean` might not give a consistent value, and it seems that the problem occurred when the argument can be evaluated to a boolean directly, i.e. `TRUE `instead of `c0=TRUE`.

Reply:
Makes sense, thought it might be useful to add something like `CREATE TABLE check_constraint_table (id int, b bool, check (b));` to give it a bit more coverage.